java-engine/README.md

```diff
@@ -118,9 +118,14 @@ The WASM library is automatically built on running tests but you'll need to make
 
 The Java engine uses FlatBuffers for communication with the WASM core. If you make changes to the data interchange format, regenerate the bindings like this:
 
+**Note:** assume flatc is available in your PATH, if not, you can specify the full path to flatc in the command below.
+
 ``` bash
+cd .. # go to yggdrasil root directory
+rustup target add wasm32-unknown-unknown
```

Comment:
I'd rather take these changes to the README out. Further up in the README, there's instructions to go complete the setup for the WASM engine, which already includes these steps. I'd rather have a single source of truth for this information so that this doesn't drift. If that needs patches can we make the changes there rather and/or try to make it clearer that you should go and do those steps first?